package.json

```diff
@@ -60,7 +60,9 @@
 		"@salesforce/ts-sinon": "^1.1.2",
 		"@types/chai": "^4",
 		"@types/cross-spawn": "^6.0.2",
+		"@types/eslint": "^8.4.1",
```

Comment:
3PP pending manual approval for some reason: https://gus.lightning.force.com/lightning/r/ADM_Third_Party_Software__c/a0qEE0000002RFRYA2/view